Formula/ghostscript.rb

```diff
@@ -78,7 +93,7 @@ def install
 
     # Install binaries and libraries
     system "make", "install"
-    system "make", "install-so"
+    ENV.deparallelize { system "make", "install-so" }
```

Comment:
Build on macOS is hitting issue here with parallel `mkdir`.
Need to confirm if this is the same location Linux is hitting issue.